pkg/e2e/operators/customdomains.go

```diff
@@ -69,16 +65,30 @@ var _ = ginkgo.Describe(customDomainsOperatorTestName, label.Operators, label.Mi
 		},
 	}
 
-	ginkgo.BeforeEach(func() {
+	var h *helper.H
+
+	ginkgo.BeforeAll(func(ctx context.Context) {
 		if viper.GetBool(config.Hypershift) {
 			ginkgo.Skip("CustomDomains Operator is not supported on HyperShift")
 		}
+
+		h = helper.New()
+		clusterversionObject, err := h.GetClusterVersion(ctx)
+		Expect(err).ShouldNot(HaveOccurred(), "unable to get cluster version")
+		clusterversion, err := semver.NewVersion(clusterversionObject.Status.History[0].Version)
+		Expect(err).ShouldNot(HaveOccurred(), "failed to build semver clusterVersion object")
+
+		if clusterversion.Major() == 4 && clusterversion.Minor() >= 13 {
```

Comment:
CDO will still be running on 4.13, but fully deprecated on 4.14
```suggestion
		if clusterversion.Major() == 4 && clusterversion.Minor() >= 14 {
```